component/node-config.jsonnet

```
@@ -0,0 +1,15 @@
+local common = import 'common.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+
+local params = inv.parameters.openshift4_nodes;
+
+local nodeConfig = kube._Object('config.openshift.io/v1', 'Node', 'cluster') {
```

I'd adjust this to also add the component common labels (see e.g. https://github.com/appuio/component-openshift4-nodes/blob/928eb622fbad18a29d8c5d6e106e61097dad5a0c/component/container-runtime.jsonnet#L13-L17):

```suggestion
local nodeConfig = kube._Object('config.openshift.io/v1', 'Node', 'cluster') {
  metadata+: {
    labels+: common.DefaultLabels,
  },
```